src/sparql/orcid-contributor-violation.sparql

```diff
@@ -0,0 +1,10 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+
+SELECT ?subject ?orcid
+WHERE {
+  ?subject dcterms:contributor ?orcid .
+  FILTER(!regex(str(?orcid), "^https://github.com/orgs/[a-z-_]+/teams/") &&
+         !regex(str(?orcid), "^(https?://orcid.org/)[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}(?:[0-9]|X)") && 
```

Comment:
I hope we remember to remove the `?` behind the `s` when new protege is rolled out..

Reply:
When I do for uberon, I'll also do it here.